Algorithm.CSharp/DelistingEventsAlgorithm.cs

```diff
@@ -179,7 +179,7 @@ public override void OnEndOfAlgorithm()
             {"Mean Population Magnitude", "0%"},
             {"Rolling Averaged Population Direction", "0%"},
             {"Rolling Averaged Population Magnitude", "0%"},
-            {"OrderListHash", "-2022527947"}
+            {"OrderListHash", "-1830199828"}
```

Comment:
😬 "Previously order was in the hash list but canceled, now it's filled but makes no difference to any of the regression statistics"?

Reply:
In this regression test case the option exercise order time was different, PR exercises ASAP while master at market close